integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,14 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--memory-swappiness", "0", "busybox", "true")
```

Comment:
@cpuguy83 , @lizf-os @hqhq with the current implementation we do not  change the existing value of container inherited by parent.
Which one is better according to you?

1. let the container inherit the value from parent (I 'll have to change the documentation too to reflect that)

2. let us override default value to 60
( This would not work with old kernels where we are not allowed to write swappiness value)